src/inmanta/app.py

```diff
@@ -81,8 +82,30 @@
 LOGGER = logging.getLogger("inmanta")
 
 
+def get_product_version() -> Optional[str]:
+    """
+    Returns the version of the Inmanta server product package.
+    This function checks for two packages: 'inmanta-service-orchestrator' and 'inmanta'.
+    It returns the version of 'inmanta-service-orchestrator' if found,
+    else it returns the version of 'inmanta'
+    """
+    try:
+        # Try to get the version of 'inmanta-service-orchestrator' (ISO)
+        return importlib.metadata.version("inmanta-service-orchestrator")
+    except importlib.metadata.PackageNotFoundError:
+        # If not found, try 'inmanta' (OSS)
+        try:
+            return importlib.metadata.version("inmanta")
+        except importlib.metadata.PackageNotFoundError:
+            return
```

Comment:
* I notice that this code is more or less a duplication of the code in the `FeatureManager` class (`FeatureManager.get_product_metadata().version`). I think we should rely on that logic instead.
* I would also move this log line into the `InmantaBootloader.start()` method. That way the log line is also present when the server is started in a different way than using the `inmanta` command.